### PR TITLE
Create new_component.md

### PR DESCRIPTION
**Pull request by micahgodbolt**
New component template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new_component.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+## Component Details
+
+#### Exports
+<!-- What will be included in index.ts file -->
+
+#### Imports
+<!-- What other components/modules will your component be using -->
+
+#### Intended Package
+<!-- OUFR/New Package/Charting etc... -->
```
Comment (dzearing) [inline .github/ISSUE_TEMPLATE/new_component.md]: If this is a prototype component, start in `@uifabric/experiments`. If you feel that there is a new package required, please indicate the requested name here.

File: .github/ISSUE_TEMPLATE/new_component.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+#### Code mockup/example
+<!-- List/describe all expected props -->
+```
+
+```
+
+#### Design Assets
+<!-- links or images -->
```
Comment (dzearing) [inline .github/ISSUE_TEMPLATE/new_component.md]: Provide screenshots of what the intended visuals for the component are.

File: .github/ISSUE_TEMPLATE/new_component.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+#### Component Ownership
+<!-- who is reponsible for bugs -->
```
Comment (dzearing) [inline .github/ISSUE_TEMPLATE/new_component.md]: Are there one or more people who can help maintain this component over time? Who will address bugs?

File: .github/ISSUE_TEMPLATE/new_component.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+---
+name: New Component
+about: Interested in contributing a new component to Fabric? This template includes necessary information to get started, and steps to completion
+
+---
+<!-- Use this template for new components or new component varients -->
+
+## Component Details
+
```
Comment (dzearing) [inline .github/ISSUE_TEMPLATE/new_component.md]: Please provide a general description of the component(s) being added.

File: .github/ISSUE_TEMPLATE/new_component.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+#### Exports
```
Comment (dzearing) [inline .github/ISSUE_TEMPLATE/new_component.md]: We probably also want a "Granularity" section which asks

Is this component broken down into granular reusable units? Consider avoiding building large, rigid components, and instead building composites based on more granular generalized ones.

File: .github/ISSUE_TEMPLATE/new_component.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+<!-- What will be included in index.ts file -->
+
+#### Imports
+<!-- What other components/modules will your component be using -->
+
+#### Intended Package
+<!-- OUFR/New Package/Charting etc... -->
+
+#### Code mockup/example
+<!-- List/describe all expected props -->
```
Comment (dzearing) [inline .github/ISSUE_TEMPLATE/new_component.md]: This probably needs an example table. It probably also needs to reference the props naming guidelines.

**Comment by micahgodbolt** [comment]
@dzearing gave a pass on the updates you suggested. Tell me what you think and we can get this merged in. Really looking forward to pointing people to this as other teams create new components.

**Pull request merged**